.eslintrc

```diff
@@ -1,6 +1,37 @@
 {
-    "extends": "vtex-react/io",
-    "rules": {
-      "react/jsx-filename-extension": 0
+  "extends": [
+    "vtex"
+  ],
+  "root": true,
+  "env": {
+    "node": true,
+    "es6": true,
+    "jest": true
+  },
+  "rules": {
+    "no-console": "off",
+    "import/order": "off",
+    "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/explicit-member-accessibility": "off",
+    "default-case": "off",
+    "global-require": "off",
+    "@typescript-eslint/no-unused-vars": [
+      "warn",
+      {
+        "argsIgnorePattern": "_+"
+      }
+    ],
+    "@typescript-eslint/interface-name-prefix": "off",
+    "max-params": "off"
```

Comment:
How about we create a card to remove the code that was getting caught by these rules and re-enable them in the future? :)